cql3/query_processor.cc

```diff
@@ -429,6 +434,11 @@ query_processor::query_processor(service::storage_proxy& proxy, database& db, se
                             [] { return authorized_prepared_statements_cache::shard_stats().authorized_prepared_statements_cache_evictions; },
                             sm::description("Counts the number of authenticated prepared statements cache entries evictions.")),
 
+                    sm::make_derive(
+                            "authorized_prepared_statements_new_gen_on_cache_size_evictions",
+                            [] { return authorized_prepared_statements_cache::shard_stats().authorized_prepared_statements_new_gen_on_cache_size_evictions; },
+                            sm::description("Counts the number of authenticated prepared statements cache entries evictions from the new generation partition due to a cache size restrictions. This is usually a sign of a cache pollution.")),
```

Comment:
Same comment I had above on the phrasing, also here.

Reply:
You are commenting about the non-existing code. Please, look at "v2: ..." patches or at the final version of files.

I deliberately left original patches so that you can review only the changes.